deploy/configs/frontend_config.py

```diff
@@ -63,15 +67,28 @@ def create_react_env_file(
         file_content = f"""GENERATE_SOURCEMAP=false
 REACT_APP_GRAPHQL_API={graphql_api_url}
 REACT_APP_SEARCH_API={search_api_url}
-REACT_APP_COGNITO_USER_POOL_ID={user_pool_id}
+REACT_APP_USERGUIDE_LINK=https://{user_guide_link}
```

Comment:
At the moment this value will be https:// right? Looking into the code the variable is only used in `frontend/src/design/components/layout/DefaultSidebar.js:201`, which is conditional on `REACT_APP_CUSTOM_AUTH`. It is very minor, but I think we can remove it and create the variable only if not custom_auth.

Reply:
Agreed, will add this change